backend/app/api/deps.py

```
@@ -38,6 +40,25 @@
         yield session
 
 
+async def get_async_db() -> AsyncGenerator[AsyncSession, None]:
+    """Get an async database session.
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)